sdk/communication/azure-communication-common/pom.xml

```diff
@@ -48,12 +48,6 @@
       <artifactId>azure-core</artifactId>
       <version>1.14.1</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
     </dependency>
-    <dependency>
-      <groupId>com.azure</groupId>
-      <artifactId>azure-core-http-netty</artifactId>
-      <version>1.9.0</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
-      <scope>compile</scope>
-    </dependency>
```

Comment:
Why is this removed? We should have netty as the default http client. 

By removing this from common, each of the other communication client libraries like chat, phone number etc will have to include this dependency.